types/sdk.js

```diff
@@ -60,7 +60,7 @@ export type AuthMiddlewareOptions = {
   scopes: Array<string>;
 }
 export type HttpMiddlewareOptions = {
-  host?: string;
+  host: string;
```

Comment:
Alright, feels like outside of the scope of this PR but will do.
Also found this: "That is, Flow understands that providing a default value makes the parameter optional. You don't need to explicitly make the parameter type optional—although it doesn't hurt."
- http://stackoverflow.com/questions/32893892/what-is-the-syntax-for-defining-a-type-when-parameter-has-a-default-value